gettsim/tests/test_tax_transfer.py

```diff
@@ -77,3 +40,44 @@ def test_tax_transfer(
         functions=policy_functions,
         targets=DEFAULT_TARGETS,
     )
+
+
+@pytest.mark.parametrize("year", YEARS)
```

Comment:
Looks good.

But should we also include a test for `_fail_if_datatype_is_false`?

Reply:
Also wir haben das bis jetzt für keine fal function und insgesamt eigl kaum uni tests. Könnte das natürlich machen, aber es ist ja auch am Anfang oft gefailed, weil ich es benutzt habe um die type hints coherent zu machen. Im Prinzip wird es ja auch durch unsere test daten gecheckt. Soll ich trotzdem?